File: app/src/main/java/co/tpcreative/supersafe/ui/move_gallery/MoveGalleryView.java
package co.tpcreative.supersafe.ui.move_gallery;
import java.util.List;

import co.tpcreative.supersafe.common.entities.ItemEntity;
import co.tpcreative.supersafe.common.presenter.BaseView;

public interface MoveGalleryView extends BaseView<ItemEntity>{
    List<ItemEntity>getListItems();
}

File: app/src/main/java/co/tpcreative/supersafe/ui/move_gallery/MoveGalleryView.kt
package co.tpcreative.supersafe.ui.move_gallery

import co.tpcreative.supersafe.common.presenter.BaseView
import co.tpcreative.supersafe.model.ItemModel

interface MoveGalleryView : BaseView<ItemModel> {
    open fun getListItems(): MutableList<ItemModel>?
}